crev-lib/src/lib.rs

```diff
@@ -133,32 +133,32 @@ pub trait ProofStore {
 }
 
 #[derive(Copy, Clone, PartialEq, Eq)]
-pub enum TrustProofType {
+pub enum ProofType {
```

Comment:
But this is not a type of any proof. It's a type of a trust proof! :) . If it at least was in a module `trust` or something ...

Reply:
That's confusing to me. Isn't this `TrustProofType` also used in creating a review proof for a crate?

I think the name "trust proof" is confusing me. I think it means "a proof for trusting an ID", like "ID proof".

What is a "trust" in crev?

Reply:
Trust Proof is one person trusting another.

There's really no "trust" in "package review proof" https://github.com/crev-dev/cargo-crev/blob/e99f7fba87c6b86324c79a7928d0bb94aa7c4689/crev-data/src/proof/review/package.rs#L61 , though people often casually say it this way "trust this crate". There's just "rating" https://github.com/crev-dev/cargo-crev/blob/e99f7fba87c6b86324c79a7928d0bb94aa7c4689/crev-data/src/proof/review/mod.rs#L29

Reply:
What I mean is that `create_review_proof(.)` uses `TrustProofType`:

https://github.com/crev-dev/cargo-crev/blob/220ff6aa567780e9d1c89cd0e6fd1892a285d1f0/cargo-crev/src/review.rs#L24

Reply:
Oh, yeah. My bad. I have re-used this type https://github.com/crev-dev/cargo-crev/blob/e99f7fba87c6b86324c79a7928d0bb94aa7c4689/cargo-crev/src/main.rs#L123 to denote when the "proof review is negative" (contains advisory) to start with negative defaults. Not a good idea.

Reply:
I understand. I've force pushed and removed my change to `TrustProofType`.